### ISSUE TITLE
[DOC] Update Pull Request Template with `netlify` option

### ISSUE THREAD

**Issue by loganthomas** [pyjanitor-devs/pyjanitor#750]
# Brief Description of Fix

Currently, the recommended approach for reviewing documentation updates is to build the docs locally. This may still be the recommended approach, but a note should be included that lets developers know that they can preview the docs from the PR checks as well (with the new `netlify` inclusion).

I would like to propose a change, such that now the docs have examples on how to use the `netlify` doc previews from a PR. 

# Relevant Context

- [Link to exact file to be edited](https://github.com/ericmjl/pyjanitor/blob/dev/.github/pull_request_template.md)
- Maybe we want to add a line to the `CONTRIBUTING.rst` files as well? 🤷‍♂️

**Comment by minchinweb** [comment]
Who has access to the Netlify preview docs? It seems like it might be a pretty short list...

**Comment by ericmjl** [comment]
@MinchinWeb I think I’m out of build minutes 😂, anyone usually can see the build with no problem.

**Comment by minchinweb** [comment]
Do you have the link to where the Netlify build would be?

**Comment by ericmjl** [comment]
@MinchinWeb yes! It usually shows up amongst the build checks. Are you able to see them?

**Comment by minchinweb** [comment]
I found them now! I guess you just have to submit a Pull Request first...
